docs/pipelines/process/resources.md

```diff
@@ -13,31 +13,363 @@ ms.date: 5/23/2019
 monikerRange: azure-devops
 ---
 
-# Resources
+# Resources in YAML
 
 **Azure Pipelines**
 
-A resource is anything used by a pipeline that lives outside the pipeline itself. Examples include:
+Any external service that is consumed as part of your pipeline is a resource.
```

Comment:
I don't think "service" is the right word here, since it includes things like secure files and service connections.

Reply:
Okay, makes sense. I have changed the wording as suggested. 

However, service connections, variable groups, secure files should not be referred as resources. We don't declare them as resources in YAML and no where in the product do we mention them as resources. If required we can have a separate doc page for `Library and configuration` so that it is clear to customer.